### PR TITLE
ci(*): fix reference to master and bash options

### PR DESCRIPTION
**Pull request by prydonius**
test-infra no longer has a reference to master so we need to fetch it to
see what's been changed

### CODE REVIEW

File: test/changed.sh [hunk 1]
```diff
@@ -13,9 +13,15 @@
...
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+git fetch --tags https://github.com/kubernetes/charts master
+
 NAMESPACE="pr-${ghprbPullId}-${BUILD_NUMBER}"
-CHANGED_FOLDERS=`git diff --find-renames --name-only ${UPSTREAM_BRANCH} | grep -v test | grep / | awk -F/ '{print $1"/"$2}' | uniq`
+CHANGED_FOLDERS=`git diff --find-renames --name-only FETCH_HEAD stable/ incubator/ | awk -F/ '{print $1"/"$2}' | uniq`
 CURRENT_RELEASE=""
```
Comment (prydonius) [inline test/changed.sh:26]: @viglesiasce is it worth adding

```
if [ -z $CHANGED_FOLDERS ]; then
  exit 0
end
```

to bypass having to initialise helm for no reason?

Comment (viglesiasce) [reply test/changed.sh:26]: SGTM

Comment (prydonius) [reply test/changed.sh:26]: done!

**Comment by viglesiasce** [comment]
Thanks @prydonius!!

**Pull request merged**